Add documentation for connectors (#10)
hevayo/ballerina-openai-client: openai.audio/Package.md

Connects to the OpenAI Audio API from Ballerina with `ballerinax/openai.audio` package.

## Package Overview
The `ballerinax/openai.audio` package is a [Ballerina](https://ballerina.io/) connector for OpenAI's Audio API. The connector provides access to state-of-the-art models for speech-to-text transcription and translation. By using the `ballerinax/openai.audio` package, you can easily integrate OpenAI's advanced AI capabilities into your own Ballerina applications.

|                             | Version         |
|-----------------------------|-----------------|
| OpenAI API                  | 1.2.0           |

## Report Issues
To report bugs, request new features, start new discussions, view project boards, etc., visit the [Ballerina Extended Library repository](https://github.com/ballerina-platform/ballerina-extended-library).

## Useful Links
- Discuss code changes of the Ballerina project via [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).

> - Join the Ballerina community on [Slack](https://ballerina.io/community/slack/) to chat with the developers.

Reviewer: We should mention our discord server in here instead

Reviewer: please update the other places as well

Connects to the OpenAI Audio API from Ballerina with `ballerinax/openai.audio` package.

## Package Overview
The `ballerinax/openai.audio` package is a [Ballerina](https://ballerina.io/) connector for OpenAI's Audio API. The connector provides access to state-of-the-art models for speech-to-text transcription and translation. By using the `ballerinax/openai.audio` package, you can easily integrate OpenAI's advanced AI capabilities into your own Ballerina applications.

> |                             | Version         |

Author: Addressed comments

|-----------------------------|-----------------|
| Ballerina Language          | Swan Lake Alpha |
| OpenAI API                  | 1.2.0           |

> 

Reviewer 2: Each connector should have a sample on how to use it in Package.md, ie How to pass the key ect. Also you can have a guide on how to generate the key in OpenAI site which will help a user starting fresh without the knowledge of OpenAI API

Author: Added the Module.md to show how to use the connectors

> Connects to the OpenAI Audio API from Ballerina with `ballerinax/openai.audio` package.

Reviewer 3: ```suggestion
Connects to the OpenAI Audio API from Ballerina with the `ballerinax/openai.audio` package.
```

> ## Package Overview

Reviewer 3: ```suggestion
## Package overview
```

The `ballerinax/openai.audio` package is a [Ballerina](https://ballerina.io/) connector for OpenAI's Audio API. The connector provides access to state-of-the-art models for speech-to-text transcription and translation. By using the `ballerinax/openai.audio` package, you can easily integrate OpenAI's advanced AI capabilities into your own Ballerina applications.

|                             | Version         |
|-----------------------------|-----------------|
| OpenAI API                  | 1.2.0           |

## Prerequisites

Before using this connector in your Ballerina application, complete the following:

> * Create an [OpenAI Account](https://beta.openai.com/signup/).

Reviewer 3: ```suggestion
* Create an [OpenAI account](https://beta.openai.com/signup/).
```

* Obtain an API key by following [these instructions](https://platform.openai.com/docs/api-reference/authentication).

> ## Quickstart

Reviewer 3: ```suggestion
## Quick start
```

To use the OpenAI Audio connector in your Ballerina application, update the `.bal` file as follows:

> ### Step 1: Import connector

Reviewer 3: ```suggestion
### Step 1: Import the connector
```

First, import the `ballerinax/openai.audio` module into the Ballerina project.

```ballerina
import ballerinax/openai.audio;
```

### Step 2: Create a new connector instance
Create and initialize `audio:Client` with the `apiKey` obtained.
```ballerina
    audio:Client audioClient = check new ({
        auth: {
            token: "sk-XXXXXXXXX"
        }
    });
```

> ### Step 3: Invoke connector operation

Reviewer 3: ```suggestion
### Step 3: Invoke the connector operation
```

> 1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.

Reviewer 3: ```suggestion
1. Now, you can use the operations available within the connector. 

>**Note:** These operations are in the form of remote operations.
```

    Following is an example on transcribing audio using the OpenAI Whisper-1 model:

>     Transcribe with Whisper-1

Reviewer 3: ```suggestion
    Transcribe with Whisper-1
```
Is this a sub-heading?

Author: https://github.com/hevayo/ballerina-openai-client/pull/10/files#r1138523502

    ```ballerina
    public function main() returns error? {
        audio:CreateTranscriptionRequest req = {
            file: {fileContent: check io:fileReadBytes("sample.mp3"), fileName: "sample.mp3"},
            model: "whisper-1"
        };
        audio:CreateTranscriptionResponse|error unionResult = check audioClient->/audio/transcriptions.post(req);
        if unionResult is audio:CreateTranscriptionResponse {
            io:println(unionResult.text);
        } else {
            io:println(unionResult);
        }
    }
    ``` 

> 2. Use `bal run` command to compile and run the Ballerina program.

Reviewer 3: ```suggestion
2. Use the `bal run` command to compile and run the Ballerina program.
```

## Report Issues
To report bugs, request new features, start new discussions, view project boards, etc., visit the [Ballerina Extended Library repository](https://github.com/ballerina-platform/ballerina-extended-library).

> ## Useful Links

Reviewer 3: ```suggestion
## Useful links
```
